terraform/groups/provisioning-role/module-iam-policies/terraform-destroy.tf

```diff
@@ -0,0 +1,47 @@
+data "aws_iam_policy_document" "destroy" {
+  statement {
+    sid       = "DestroyPolicyListedResourcesWithConditions"
+    effect    = "Allow"
+    resources = [
+      "arn:aws:ec2:eu-west-2:${var.aws_account_id}:security-group/*"
```

Comment:
Is it possible to limit the resource to `arn:aws:ec2:eu-west-2:${var.aws_account_id}:security-group/payment-reconciler-*`
? This would remove the need for the condition

Reply:
No, unfortunately not. I did try but security group ARNs are the ID (hence random number) not the name.